skuba-update/Makefile

```diff
@@ -1,17 +1,5 @@
 .PHONY: all
-all: install
-
-.PHONY: install
-install:
-	./setup.py install
```

Comment:
my question here, is this file still needed? I am not super famillier with the project but it did not seem like this was used elsewhere? can we also delete setup.py while we are at it?